packages/fuel-gauge/src/vectors.test.ts

```diff
@@ -119,7 +120,7 @@ describe('Vector Tests', () => {
   it('should test (u8, u8) vector input/output', async () => {
     using contractInstance = await setupContract();
 
-    const INPUT: Array<[BigNumberish, BigNumberish]> = [
+    const INPUT: Vec<[BigNumberish, BigNumberish]> = [
```

Comment:
We should rethink the need for the `Vec` helper type as it's essentially the same as `Array<T>` or `T[]`.

Reply:
Agreed - added [here](https://github.com/FuelLabs/fuels-ts/issues/3032).